### ISSUE TITLE
thoughts on adding a file for globals like "game" or "newproxy"?

### ISSUE THREAD

**Issue by zainthemaynnn** [CloneTrooper1019/Roblox-Client-Watch#20]
I couldn't find them anywhere here. or, anywhere at all, for that matter. for now I made a web scraper for [this page](https://developer.roblox.com/en-us/api-reference/lua-docs/Roblox-Globals) as a Rust project. one contains the globals themselves, the other one is an enhanced version of the original autocomplete file. it might be useful to host this here, but I didn't know if it fits or not since I think it's a bit of a hacked together solution anyways. here's a snap of both files.
![image](https://user-images.githubusercontent.com/69121865/123486207-9804c100-d5d9-11eb-9a87-36285167c382.png)
![image](https://user-images.githubusercontent.com/69121865/123486321-cc787d00-d5d9-11eb-86e7-84130ae75edd.png)
if not, I can hold this in separate repository. thanks.

edit: it can also work on [this page](https://developer.roblox.com/en-us/api-reference/lua-docs/Lua-Globals) which isn't supported by the autocomplete file either.

**Comment by MaximumADHD** [comment]
This would be worth filing as a feature request on Roblox’s DevForums. The AutocompleteMetadata.xml file is directly ripped from Roblox Studio without any modifications or enhancements made.

**Comment by MaximumADHD** [comment]
As a matter of fact the file only exists to add context to built-in functions, not to provide a full list of built-in keywords. You’d be better off just hard-coding some additional references like these instead of relying on one source of truth, since it’s likely those fields will never change or be removed anyway.

**Comment by zainthemaynnn** [comment]
sure, that makes sense. I attempted to request this a while ago but unfortunately I don't have permissions to post to that board :)

as for the hardcoding, I've also seen this done in a few other projects and I really think there should be some definitive place to get this data. in my case I also needed descriptions. not enough to be tedious, but I still found it weird that it doesn't already exist. I doubted you would be interested in the xml file in the first place, this was more about the first one.

however I just checked studio and those globals surprisingly don't have any descriptions associated with them. I guess that's why they're not in here... but I'm not sure what the reasoning is for that. I'll probably ask if I attain regular status someday. thanks.
